backend/src/apiserver/server/api_converter_test.go

```diff
@@ -101,6 +106,22 @@ func TestToApiRunDetail(t *testing.T) {
 		PipelineRuntime: model.PipelineRuntime{WorkflowRuntimeManifest: "workflow123"},
 	}
 	apiRun := ToApiRunDetail(modelRun)
+
+	listParams := []interface{}{1, 2, 3}
+	v2RuntimeListParams, _ := structpb.NewList(listParams)
+
+	structParams := map[string]interface{}{"structParam1": "hello", "structParam2": 32}
+	v2RuntimeStructParams, _ := structpb.NewStruct(structParams)
+
+	// Test all parameters types converted to model.RuntimeConfig.Parameters, which is string type
+	v2RuntimeParams := map[string]*structpb.Value{
+		"param2": structpb.NewStringValue("world"),
```

Comment:
nit: why starting with 2?

Reply:
I'm not sure why, but all test cases in `api_converter_test.go` have parameters starting at 2.

Reply:
Not a big deal yet it just reads odd. I really don't see why this needs to be repeated or reinforced.